Makefile

```diff
@@ -425,7 +457,7 @@ STERN_VERSION = v2.6.1
 CHART_TESTING_VERSION = v3.11.0
 K3D_IMAGE = docker.io/rancher/k3s:v1.31.1-k3s1
 TESTS = [nginx,api,features-kubernetes,bulk-deployment,features-kubernetes-2,features-variables,active-standby-kubernetes,tasks,drush,python,gitlab,github,bitbucket,services,workflows]
-CHARTS_TREEISH = main
+CHARTS_TREEISH = keycloak-tls-updates
```

Comment:
now https://github.com/uselagoon/lagoon-charts/pull/733 has been merged this can be set back

Reply:
The UI also has an override that will need to be reverted at some stage https://github.com/uselagoon/lagoon/pull/3860/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R58